Extend German translation

wlhlm opened the pull request:
Adding a few bits that were missing.

Vexatos commented on src/main/resources/assets/openblocks/lang/de_DE.lang:
```diff
@@ -70,13 +86,24 @@ openblocks.misc.selected_cannon=Kanone zum Zielen ausgewählt
-openblocks.misc.stored_inventory=Inventar des Spielers gespeichert in %s
+openblocks.misc.page=Seite %d von %d
+openblocks.misc.oh_no_ceiling=Du kannst hier nicht schlafen. Die Decke beunruhigt dich zu sehr...
+openblocks.misc.oh_no_ground=Auf dem DING willst du schlafen?!
```
How about having it "Auf DEM Ding willst du schlafen ?!" Sounds a bit better, in my opinion.

wlhlm replied:
Yeah, makes more sense.

wlhlm commented:
Updated based on feedback.

XFactHD commented on src/main/resources/assets/openblocks/lang/de_DE.lang:
```diff
@@ -24,20 +35,25 @@ openblocks.gui.itemdropper=Itemspender
 openblocks.gui.digitalfuse=Digitalzünder
+openblocks.gui.xp_outputs=XP-Abgabe
+openblocks.gui.item_outputs=Itemabgabe
+openblocks.gui.watch_video=Sehe Video
```
Should be "Siehe Video".

XFactHD commented on src/main/resources/assets/openblocks/lang/de_DE.lang:
```diff
@@ -24,20 +35,25 @@ openblocks.gui.itemdropper=Itemspender
 openblocks.gui.config.content=OpenMods erlaubt, sowohl Client- als auch Serverkonfigurationen über den Chat zu ändern. Manche brauchen nicht mal einen Neustart!\n\nServerkonfigurationen können mit '§L/om_config_s§R' (setzt Operator-Rechte vorraus) und Clientkonfigurationen mit '§L/om_config_c§R' geändert werden.\n\nZum Beispiel:\n\n§L/om_config_s set OpenBlocks dropblock elevatorDrainsXP false§R\ ...
 
+openblocks.gui.restore_inv.title=Inventar-Backup
+openblocks.gui.restore_inv.content=Jeder kann einen Fehler machen, aber mit dem §L/ob_inventory§R-Befehl kannst du einige von ihnen wieder beheben. Dieser Befehl kann zum Wiederherstellen des Inventars genutzr werden. Ein Backup wird erstellt wenn:\n - der Spieler stirbt\n - ein Grabstein gespawnt wird \n - der Spieler den Befehl §L/ob_inventory store§R eingibt\n\nTo restore an inventory you need ...
```
Should be "genutzt".

XFactHD commented on src/main/resources/assets/openblocks/lang/de_DE.lang:
```diff
@@ -70,13 +86,24 @@ openblocks.misc.selected_cannon=Kanone zum Zielen ausgewählt
+openblocks.misc.invalid_sub_inventory=Kann Unterinventar %s nicht widerherstellen
+openblocks.misc.stored_inventory=Spielerinventar gespeichert in %s
 openblocks.misc.restored_inventory=Inventar von Spieler %s wiederhergestellt
+openblocks.misc.invalid_index=Invalider index
```
Should be "Fehlerhafter Index".

XFactHD commented on src/main/resources/assets/openblocks/lang/de_DE.lang:
```diff
@@ -135,18 +171,29 @@ openblocks.turtle.magnet=Magnet
+tile.openblocks.guide.video=http://www.youtube.com/watch?v=IxSZ-mOFHX0
+
+tile.openblocks.builder_guide.name=Verbesserte Bau-Hilfe
+tile.openblocks.builder_guide.description=Die verbesserte Form der Bau-Hilfe zeigt nicht nur Geist-Blöcken zum einfacheren Bauen, sondern erlaubt auch das Setzen der eigentlichen Blöcke. Klicke dazu einfach auf den zentralen Block mit einem beliebigen Stapl!\nIm Kreativmodus kannst du einen Obisidanblock darüber platzieren und dann die Bau-Gilfe mit einem beliebigen Block hauen um das Gebäude zu  ...
```
Should be "Phantomblöcke", "Stapel" and "Bau-Hilfe".

XFactHD commented on src/main/resources/assets/openblocks/lang/de_DE.lang:
```diff
@@ -135,18 +171,29 @@ openblocks.turtle.magnet=Magnet
-tile.openblocks.ladder.name=Jaded-Leiter
-tile.openblocks.ladder.description=Die Jaded-Leiter löst das nervige Problem, über eine Falltür an der Spitze einer Leiter springen zu müssen. Die Jaded-Leiter funktioniert zeitgleich wie eine Falltür und wie eine Leiter.
+tile.openblocks.ladder.name=Erschöpfte Leiter
+tile.openblocks.ladder.description=Die Erschöpfter Leiter löst das nervige Problem, über eine Falltür an der Spitze einer Leiter springen zu müssen. Die Erschöpfte Leiter funktioniert zeitgleich wie eine Falltür und wie eine Leiter.
```
Should be "Strickleiter" instead of "Erschöpfte Leiter"

wlhlm commented:
Updated.

Thanks for the feedback!

wlhlm commented:
Updated again.

Removed the video links as per 98f41070a6f159d94695c3ea7b8fb8cb29984f88.

Please let me know what else is holdings this back from merging.

boq commented:
Nothing. I just wait for release to merge it, to limit amount of PRs.

wlhlm commented:
I see. I saw the other open translation PRs and was a little worried.
